_data/bwuzhang/bwuzhang.md

```diff
@@ -0,0 +1,7 @@
+---
+name: bwuzhang 🐶
```

Comment:
Please follow the given [instructions](https://github.com/education/graduation#3-add-your-profile-information) by removing the emoji to maintain the same format

Reply:
is emoji not allowed? there's a flag emoji in on the institution line on the official template

Reply:
While reviewing PR, I have not come across a single PR who has emoji on the name tag and I personally don't see a problem wiith it. Maybe you can try keeping it and see if it gets merged. ;)

Reply:
According to this https://github.com/education/graduation/issues/348#issuecomment-631776551 There's no problems with emojis.